src/status_im/ui2/screens/chat/components/edit/style.cljs

```diff
@@ -0,0 +1,26 @@
+(ns status-im.ui2.screens.chat.components.edit.style)
+
+(def container
+  {:flex-direction      :row
+   :height              24
+   :accessibility-label :edit-message})
```

Comment:
hmm I don't think `accessibility-label` is part of `style` prop.
imo things that go in style.cljs files should not be anything outside of - style props https://reactnative.dev/docs/view-style-props

So in your component it should be:

```
style.cljs
(def container
  {:flex-direction      :row
   :height              24})
   
  view.cljs 
[rn/view {
 :style style/container
 :accessibility-label :edit-message
}
```
cc @OmarBasem , @ilmotta , @flexsurfer wdyt?

Reply:
yes its not, it should be outside styles

Reply:
Of course you are right, left over, will remove it